src/components/pages/home/elements/weatherAttributes/TemperatureElement.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+export default class TemperatureElement extends Component {
+  render() {
+    return (
+      <div className="temperatureElement">
+        <h2>Temperature</h2>
+        <h3>Avg: {this.props.Temp.avg} {'\u2103'} </h3>
```

Comment:
I *think* this can be written using string interpolation as ``{`${this.props.temp.avg} \u2103`}`` to avoid having to use multiple JS blocks.